config/locales/qqq.yml

```diff
@@ -242,6 +242,10 @@ qqq:
     milestones:
       title: Label for the Milestone section of a course page (Wiki Ed only)
   campaign:
+    active_campaigns: |-
+      Heading for the list of active campaigns. 
```

Comment:
This should be just "Heading for the list of active campaigns". The rest can be removed.